input/fsh/DEF_Rulesets.fsh

```diff
@@ -1,13 +1,15 @@
 
-// Common for all RT Requests including Course Summary
+// Common for all RT Requests
 RuleSet: RadiotherapyRequestCommon
 // * meta MS
 // * meta.versionId MS
 // * meta.lastUpdated MS
 * extension MS
 * extension contains
-    $mCODERadiotherapyModalityAndTechnique named modality-and-technique 0..* MS and
-    RadiotherapyDosePrescribedToVolume named radiotherapy-dose-prescribed-to-volume 0..* MS
+    $mCODERadiotherapyModalityAndTechnique named modalityAndTechnique 0..* MS
```

Comment:
I think it may be better to define a new Codex-RT extension that is a profile of the mCODE modality and technique extension.   See http://build.fhir.org/ig/HL7/codex-radiation-therapy/branches/add-plan-summaries-sak/StructureDefinition-codexrt-radiotheraphy-phase-prescription.html

I think it is better because:
   - the defining URL of the new extension makes it clear that it is different than the mCODE extension
   - it preserves all of the invariants
   - there is a clickable entity in the IG that shows the content of the extension....you still have to click through it to see the content of the mCODE extension, but I think it looks cleaner.
     - I've asked Mark for his opinion.

Reply:
The invariants would still be preserved if we'd use the original mCODE extension. I see the advantange of having a dedicated url for the complex extension that has everything (modality, technique, device type, energy or isotope (to be defined) and maybe more in the future). But since we don't have good name for it I was not so keen on that. On the other hand just having new extensions device-type, energy, isotope and not care about the context in the first place would be very simple as we discussed already. We just happen to use them here in the modality-and-technique hierarchy but in principle they would be self standing extensions for any context. I understand we could have both, self standing extensions for all elements without restricted context AND use them in a complex extension derived from the mCODE extension . But the simplest to me is still just new single element extensions and the resource profile just put them in the right place for these profiles, i.e. in the modality and technique mCODE extension.

Reply:
I'm pretty much convinced that status quo is fine....still waiting for Mark to weigh in.